app/static/src/components/run/RunTab.vue

```diff
@@ -149,6 +152,18 @@ export default defineComponent({
             xAxis.value = newAxis;
         };
 
+        onMounted(() => {
+            if (props.visibleVars) {
+                const visibleVars = props.visibleVars.split(",").map(s => s.trim());
+                graphConfigs.value.forEach((_, graphIndex) => {
+                    store.dispatch(`graphs/${GraphsAction.UpdateSelectedVariables}`, {
+                        graphIndex,
+                        selectedVariables: visibleVars
+                    });
+                });
```

Comment:
This is potentially overwriting saved graph configs though, isn't it? Are we not worrying about that because we don't expect multiple graph config settings to be saved in static wodin? in that case should we more explicitly use visibleVars if static and graphConfig selectedVariables if dynamic? Feels like a real conflict.